common/djangoapps/pipeline_mako/templates/mako/inline_js.html

```diff
@@ -1,9 +1,12 @@
+<%page expression_filter="h"/>
+<%! from openedx.core.djangolib.markup import HTML %>
 <script \
-% if async:
+% if context.get('async', default=None):
 async \
 % endif
-if defer:
+% if defer:
```

Comment:
I think that this change is needed here, which makes me think that this file is not being used.